### PR TITLE
creatign cars2.js

### PR DESCRIPTION
**Pull request by mekhamata**
added the cars2 variable and it's awesome works great!

### CODE REVIEW

**Comment by mikha-m** [comment]
this need to be fixed

File: cars2.js [hunk 1]
```diff
@@ -0,0 +1 @@
+const cars2 = "this is my cars 2";
```
Comment (mikha-m) [inline cars2.js:1]: it needto be CAR

Comment (mikha-m) [inline cars2.js:1]: Starting review...

**Review by mikha-m — CHANGES_REQUESTED** [review]
Please fix this

File: cars2.js (hunk 1 above)
Comment (mekhamata) [reply cars2.js:1]: problem fixed

**Review by mikha-m — APPROVED** [review]
approved

**Pull request merged**